README.md

```diff
@@ -293,6 +293,8 @@ An array of pull request labels that are forbidden from pull request review.
 * `PULL_REVIEW_CONFIG_PATH`: location of the config file in the pull request repo (default is `.pull-review`).
 * `PULL_REVIEW_CONFIG`: Pull Review configuration override in JSON/YAML format.
 * `PULL_REVIEW_REQUIRED_ROOMS`: whitelist of Hubot chat rooms for Pull Review requests (e.g. `dev,ops`).
+* `PULL_REVIEW_NOTIFY_SLACK`: send message to slack (default `true`)
```

Comment:
this is better expressed in .pull-review config, e.g. https://github.com/imsky/pull-review/blob/b7ec86367c5dd8da47bbee719a8272fbb0220433/.pull-review#L35-L36

env vars are used for config that's complicated to put in the .pull-review file

there are a couple of alternatives here:

1. just allow disabling GitHub notifications, e.g. `notify_github: false`
2. add a list of notification channels, e.g.
```yaml
notification_channels:
  - slack
  - github
```